arkane/encorr/ae.py

```diff
@@ -125,6 +131,10 @@ def execute(self, output_file: str = None):
                 f.writelines(self.ae.format_atom_energies(
                     'atom_energies' if self.level_of_theory is None else self.level_of_theory))
 
+            if plot:
+                correlation_path = os.path.join(output_directory, f'AEC_{model_chemistry_formatted}_correlation.pdf')
```

Comment:
add a unique timestamp to the end so you don't overwrite previous files by accident

Reply:
I don't think this is necessary though because we put the level of theory name in the save path (`model_chemistry_formatted` ). Say if you fit for the same level of theory and plot it for multiple times because something wasn't quite right previously, I think as a user you would only care about the last one.

Reply:
RE in-person discussion: update arkane docs to state that previous files will be overwritten on subsequent runs